packages/preview/codly/0.1.0/example/main.typ

```diff
@@ -0,0 +1,98 @@
+#import "../codly.typ": *
+
+#show: codly-init.with()
+
+#let icon(codepoint) = {
+  text(
+      font: "tabler-icons", 
+      fallback: false, 
+      weight: "regular", 
+      size: 8pt,
+  )[#codepoint]
+}
```

Comment:
Probably add this to readme? My first thought, when I saw this in readme, was that the `icon` was defined in the package. Maybe also a good idea to add the font too